includes/base/element-base.php

```diff
@@ -116,7 +116,7 @@ public function get_controls( $control_id = null ) {
 		return self::_get_items( $stack['controls'], $control_id );
 	}
 
-	public function add_control( $id, $args ) {
+	public function add_control( $id, array $args ) {
```

Comment:
Hardly.

Adding type hinting could break existing code if it's added late in the project.

Not saying it's a bad thing to have, but surely isn't a thing that should appear in _docs_ commits :)